docs/static/security/tls-encryption.asciidoc

```diff
@@ -1,8 +1,8 @@
 [discrete]
 [[ls-http-ssl]]
-=== Configuring Logstash to use TLS encryption
+=== Configuring Logstash to use TLS/SSL encryption
 
-If TLS encryption is enabled on the {es} cluster, you need to
+If TLS encryption is enabled on an on premise {es} cluster, you need to
```

Comment:
The previous section about "configure the elasticsearch output" covers setting up the SSL / cacert. We should find a way to combine so that we only talk about it in one location.

Reply:
Tracked in #13620